summary/table.go

```diff
@@ -100,42 +100,42 @@ func DisplaySummaryOutput(resp *opb.Response, revDNSLookup bool, requestType str
 	}
 
 	if strings.Contains(requestType, "network") {
-		if len(resp.InNwData) > 0 {
+		if len(resp.IngressConnection) > 0 {
 			fmt.Printf("\nIngress connections\n")
 			// Display server conn data
 			inNwRowData := [][]string{}
-			for _, inNwData := range resp.InNwData {
+			for _, IngressConnection := range resp.IngressConnection {
 				inNwStrSlice := []string{}
-				domainName := dnsLookup(inNwData.IP, revDNSLookup)
-				inNwStrSlice = append(inNwStrSlice, inNwData.Protocol)
-				inNwStrSlice = append(inNwStrSlice, inNwData.Command)
+				domainName := dnsLookup(IngressConnection.IP, revDNSLookup)
+				inNwStrSlice = append(inNwStrSlice, IngressConnection.Protocol)
+				inNwStrSlice = append(inNwStrSlice, IngressConnection.Command)
 				inNwStrSlice = append(inNwStrSlice, domainName)
-				inNwStrSlice = append(inNwStrSlice, inNwData.Port)
-				inNwStrSlice = append(inNwStrSlice, inNwData.Namespace)
-				inNwStrSlice = append(inNwStrSlice, inNwData.Labels)
-				inNwStrSlice = append(inNwStrSlice, inNwData.Count)
-				inNwStrSlice = append(inNwStrSlice, inNwData.UpdatedTime)
+				inNwStrSlice = append(inNwStrSlice, IngressConnection.Port)
+				inNwStrSlice = append(inNwStrSlice, IngressConnection.Namespace)
+				inNwStrSlice = append(inNwStrSlice, IngressConnection.Labels)
+				inNwStrSlice = append(inNwStrSlice, IngressConnection.Count)
+				inNwStrSlice = append(inNwStrSlice, IngressConnection.UpdatedTime)
 				inNwRowData = append(inNwRowData, inNwStrSlice)
 			}
 			WriteTable(SysNwHeader, inNwRowData)
 			fmt.Printf("\n")
 		}
 
-		if len(resp.OutNwData) > 0 {
+		if len(resp.EgressConnection) > 0 {
 			fmt.Printf("\nEgress connections\n")
 			// Display server conn data
 			outNwRowData := [][]string{}
-			for _, outNwData := range resp.OutNwData {
+			for _, EgressConnection := range resp.EgressConnection {
 				outNwStrSlice := []string{}
```

Comment:
here also we have outNwData and OutNwData. Be cautious about the casing :grin:

Reply:
Changes are done. Thank you